transaction-pool/src/scoring.rs

```diff
@@ -138,7 +138,7 @@ impl<T, S: Clone> Clone for ScoreWithRef<T, S> {
 impl<S: cmp::Ord, T> Ord for ScoreWithRef<T, S> {
 	fn cmp(&self, other: &Self) -> cmp::Ordering {
 		other.score.cmp(&self.score)
-			.then(other.transaction.insertion_id.cmp(&self.transaction.insertion_id))
+			.then(self.transaction.insertion_id.cmp(&other.transaction.insertion_id))
```

Comment:
Am I reading this right? First compare the scores; if equal then compare the insertion_ids and pick the lower (older?) one. That way in a no-gas scenario txs are processed oldest-first? Maybe a comment is warranted here?

Reply:
Correct, the tests are documenting that behaviour.